src/data-source/DataSource.ts

```diff
@@ -760,4 +756,129 @@ export class DataSource {
         }
         return "slave"
     }
+
+    /**
+     * Run fn with a temporary query runner.
+     *
+     * Query runner is automatically released after fn is done.
+     *
+     * @param fn
+     */
+    async runWithQueryRunner<T>(
+        fn: (queryRunner: QueryRunner) => Promise<T>,
```

Comment:
What is fn?

Reply:
https://github.com/typeorm/typeorm/pull/11197/files#diff-bc29c8a16d80c175364ccb4e84a5247cc0cf5ab6d2aff241a8a871084a089300 I added it to the docs. Internally the overloaded version that takes the optional first `queryRunner` argument is more useful, e.g:

https://github.com/typeorm/typeorm/pull/11197/files#diff-06402bac7d3c217ce8afc1071fffcd25d026b57b4f0c19593615be01539359b4R147-R164

Reply:
is there maybe a better suited name instead of `fn` that provides more intent?